src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1291,15 +1291,17 @@ To be able to edit the default logo displayed in ProActive portals, you have to
 
 == User Authentication
 
-In order to use ProActive Scheduler every user must have an account. It supports three methods for authentication:
+In order to log into ProActive server, every user must have a registered account.
+ProActive Scheduler and Resource Manager support several methods to authenticate users:
 
-* *File based*
-* *LDAP*
-* *PAM*
+* <<_file,*File-based*>>
+* <<_ldap,*LDAP*>>
+* <<_ldap_with_multiple_domains,*LDAP with Multiple Domains*>>
+* <<_pam,*PAM*>>
 
 === Select authentication method
 
-By default the ProActive Scheduler is configured to use *file based* authentication and has some default accounts ('demo/demo', 'admin/admin') that
+By default, the ProActive server is configured to use *file based* authentication and has some default accounts ('demo/demo', 'admin/admin') that
```

Comment:
Suggest to remove:
- server
- user passwords

By default, ProActive is configured to use *file based* authentication and has some default accounts ('demo', 'admin') that ..